export.py

```diff
@@ -2350,6 +2352,9 @@ def export_render_settings(ri, rpass, scene, preview=False):
     ri.PixelFilter(rm.pixelfilter, rm.pixelfilter_x, rm.pixelfilter_y)
     ri.ShadingRate(rm.shadingrate)
     ri.Attribute("trace", depths)
+    if rm.use_separate_path_depths:
+        ri.Option("trace", {'string depthmode': 'separate'})
```

Comment:
So first of all you're going to output an option multiple times postentially.  Which doesn't really matter here but generally a weird idea.  And options should really be output earlier in the rib with the others....  Maybe when options are output could scan through and see if any have relative pixel variance set?